braddeibert/musclenerds
# Workout catalog pt. 1

CHANGES:
* Implements the workout catalog screen as per prototype
* Pulls all workouts from the database and lists them under corresponding muscle group worked
* Tapping on a workout opens a workout view fragment, that shows very basic info about the workout 

TESTING:
* Navigate to the workout catalog in the app
* Be sure all workouts from database are showing in cards, under their respective muscle group (cross check using Android Studio's database inspector)
* Click a workout and see the bug, lol (you should see it load info about the workout you chose on top of the catalog)

KNOWN BUG:
* Workout catalog fragment is not replaced when selecting a workout (i.e. the catalog and selected workout fragments overlap)
